ydb/core/tx/schemeshard/schemeshard__operation_modify_acl.cpp

```diff
@@ -51,6 +51,24 @@ class TModifyACL: public TSubOperationBase {
             return result;
         }
 
+        if (acl) {
+            NACLib::TDiffACL diffACL(acl);
+            for (const NACLibProto::TDiffACE& diffACE : diffACL.GetDiffACE()) {
+                if (static_cast<NACLib::EDiffType>(diffACE.GetDiffType()) == NACLib::EDiffType::Add) {
+                    if (!context.SS->LoginProvider.Sids.contains(diffACE.GetACE().GetSID())) {
+                        result->SetError(NKikimrScheme::StatusPreconditionFailed, "Sid not found");
```

Comment:
```suggestion
                        result->SetError(NKikimrScheme::StatusPreconditionFailed, "Sid is not found");
```

Reply:
I'm suspicious about that change as in lots of other places we use a short form

Grammatically incorrect, but who cares

https://github.com/search?q=repo%3Aydb-platform%2Fydb%20%22not%20found%22&type=code

https://github.com/ydb-platform/ydb/blob/892a954b648dfce3da44ad433b2258e303b08b06/ydb/library/login/login.cpp#L100